src/7-to-8/major-changes/cli.rst

```diff
@@ -27,61 +27,79 @@ renamed or removed.
 ``cylc documentation``
   - We no longer include a command for locating this documentation.
 ``cylc edit``
   - Use a text editor to edit the workflow configuration file.
-``cylc insert``
-  - Task insertion is now automatic, use ``cylc trigger``.
 ``cylc jobscript``
   - It is no longer possible generate a jobscript from outside of a workflow.
-``cylc monitor``
-  - There is now a new more powerful terminal user interface (TUI).
-  - Try ``cylc tui``.
 ``cylc nudge``
   - No longer required.
-``cylc print``
-  - Equivalent to ``cylc scan --states=all``.
 ``cylc register``
   - Registration is no longer required, all workflows in the ``~/cylc-run``
     directory are "registered" automatically.
   - To install a workflow from a working copy use ``cylc install``.
-``cylc reset``
-  - It is no longer possible to manually change a task's state.
-  - You can, however, override the outputs the task generated which has a
-    similar effect with ``cylc set-outputs``.
-``cylc restart``
-  - ``cylc run``, ``cylc pause`` and ``cylc release`` have been combined into
-    ``cylc play``.
 ``cylc review``
   - The read-only ``cylc review`` web GUI has been removed.
   - The latest Cylc 7 version of ``cylc review`` is Cylc 8 compatible
     so can still be used to monitor both Cylc 7 and Cylc 8 workflows
     side by side.
-``cylc run``
-  - ``cylc run``, ``cylc pause`` and ``cylc release`` have been combined into
-    ``cylc play``.
 ``cylc search``
   - Use ``grep`` or a text editor to search the workflow configuration or
     source directory.
-``cylc spawn``
-  - Spawning is now performed automatically, use ``cylc trigger`` to run a task.
 ``cylc submit``
   - It is no longer possible to submit a job from outside of a workflow.
 ``cylc warranty``
   - The Cylc license remains unchanged from Cylc 7.
 
+.. rubric:: Commands that have been replaced:
+
+``cylc conditions``
+  - See the `license`_ file for conditions of usage, or ``cylc help license``
+  - The Cylc license remains unchanged from Cylc 7.
+``cylc get-config``,
+  - Replaced by ``cylc config``.
+``cylc get-*-config``
+  - (Where ``*`` is ``site``, ``suite`` or ``global``)
+  - Replaced by ``cylc config``.
+``cylc graph-diff``
+  - Replaced by ``cylc graph <flow1> --diff <flow2>``
+``cylc insert``
+  - Task insertion is now automatic, use ``cylc trigger``.
+``cylc monitor``
+  - There is now a new more powerful terminal user interface (TUI).
+  - Try ``cylc tui``.
+``cylc print``
+  - Equivalent to ``cylc scan --states=all``.
+``cylc reset``
+  - It is no longer possible to manually change a task's state.
+  - You can, however, override the outputs the task generated which has a
+    similar effect with ``cylc set-outputs``.
+``cylc restart``
+  - Replaced by ``cylc play``.
+``cylc run``
+  - Replaced by ``cylc play``.
+``cylc spawn``
+  - Spawning is now performed automatically, use ``cylc trigger`` to run a task.
+``cylc suite-state``
+  - Renamed as ``cylc workflow-state``.
```

Comment:
```suggestion
  - Renamed to ``cylc workflow-state``.
```

Reply:
["Renamed as" seems to be more favoured](https://books.google.com/ngrams/graph?content=renamed+as%2Crenamed+to&year_start=1800&year_end=2019&corpus=26&smoothing=3&direct_url=t1%3B%2Crenamed%20as%3B%2Cc0%3B.t1%3B%2Crenamed%20to%3B%2Cc0)

Reply:
Actually, I agree, on reflection.